docusaurus.config.js

```diff
@@ -0,0 +1,134 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: 'Jellyfish docs',
+  tagline: 'Jellyfish are cool',
+  favicon: 'img/membrane-signet.svg',
```

Comment:
We should probably use the newer logo, the one with light blue background. Also, this one looks really bad in dark mode because of transparency.

Reply:
Fixed it